kernel/byterun/coq_interp.c

```diff
@@ -1454,6 +1454,44 @@ value coq_interprete
         Next;
       }
 
+      Instruct(CHECKDIVSINT63) {
+        print_instr("CHEKDIVSINT63");
+        CheckInt2();
+        /* spiwack: a priori no need of the NON_STANDARD_DIV_MOD flag
+                    since it probably only concerns negative number.
+                    needs to be checked at this point */
```

Comment:
Is this comment really wanted?

Reply:
I guess there's no reason for it to appear two times.